### DISCUSSION TITLE
Add a profile for bottles

### DISCUSSION THREAD

**Discussion by Raj2032** [netblue30/firejail#6512] [Q&A]
without using firejail I installed notepad++. After closing bottles and opening it up again through firejail via the command:

```bash
firejail bottles
```

without setting up any further restrictions I was able to launch bottles however when launching notepad++ I get the following error through terminal:

```bash
22:16:48 (INFO) Using Wine Starter -- run 
wine: chdir to /tmp/.wine-1000/server-802-4c1378 : Permission denied
^C
Parent received signal 2, shutting down the child process...

Child received signal 2, shutting down the sandbox...

Parent is shutting down, bye...
```

not too sure how to fix this, I think `/tmp` dir is blacklisted?

**Comment by Raj2032** [comment]
@rusty-snake hey mate this is actually not a duplicate this is a different issue now, before I was asking regarding if I used firefox profile for this, now I am just using default configuration.

**Reply by rusty-snake** [reply]
It's a duplicate of #1139 (https://github.com/netblue30/firejail/issues/1139#issuecomment-1218227816) because there is no profile for it, so it is effective a profile request.

**Reply by Raj2032** [reply]
@rusty-snake Ah right my mistake

**Reply by Raj2032** [reply]
@rusty-snake I suppose I should just stick to the flatpak version instead then and use flatseal and stuff or is there a workaround using firejail?
